mt-kahypar/partition/coarsening/nlevel_uncoarsener.cpp

```diff
@@ -147,7 +158,7 @@ namespace mt_kahypar {
           total_batches_size += batch.size();
           // Update Progress Bar
           uncontraction_progress.setObjective(current_metrics.getMetric(
-              _context.partition.mode, _context.partition.objective));
+              translateMode(_context.partition.mode), _context.partition.objective));
```

Comment:
Hm I didn't add it because this is the only file where the kahypar mode is actually used. However, I noticed now that at all other location where `Metrics` is used, `kahypar::Mode::direct_kway` is just passed directly as parameter.

Also, `Metrics` isn't actually doing anything else besides holding a given value, so I wonder whether we actually need to import it from kahypar at all?